core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -7,6 +9,16 @@ use zksync_types::{
 
 use crate::Tokenizable;
 
+/// Loads KZG settings from the file system.
+pub fn load_kzg_settings() -> KzgSettings {
+    static KZG_SETTINGS: Lazy<KzgSettings> = Lazy::new(|| {
+        let zksync_home = std::env::var("ZKSYNC_HOME").unwrap_or_else(|_| ".".into());
+        let path = std::path::Path::new(&zksync_home).join("trusted_setup.json");
+        KzgSettings::new(path.to_str().unwrap())
+    });
+    KZG_SETTINGS.clone()
+}
```

Comment:
@popzxc what should we do with? 
seems we have to provide the whole path

Reply:
Responded elsewhere. I think it's fine to depend on `ZKSYNC_HOME`, just in a place that is aware of `ZKSYNC_HOME` anyways.